README.md

```diff
@@ -132,6 +132,8 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | `NUM_WORKERS`                 | number of VMs to spawn as workers, default: 0 |
 | `OPENSHIFT_VERSION`           | OpenShift version to install, default: "4.7" |
 | `HYPERTHREADING`              | Set node's CPU hyperthreading mode. Values are: all, none, masters, workers. default: all |
+| `DISK_ENCRYPTION_MODE`        | Set disk encryption mode. Right now assisted-test-infra only supports "tpmv2". |
```

Comment:
If we use tpmv2 as default then we should mention what is the default (I am not sure we want a default in that case though)

Reply:
I see
I can add a new mode ``"none"`` that will be a default, and that keeps this object empty then. I'll implement that when your PR is in
Also I added one comment on that PR, which I'm not sure is an issue but I'll let you check it